src/temp-types-based-on-js-source.ts

```diff
@@ -34,6 +34,8 @@ export interface ESTreeNode {
   value?: string;
   expression?: ESTreeNode | null;
   decorators?: (ESTreeNode | null)[];
+  implements?: ESTreeNode[];
+  extends?: ESTreeNode[];
```

Comment:
extends can be `array of nodes` or `node`

extends in class is node, in interface is array

Reply:
It's superClass in classes right? Extends isn't used?

Reply:
ahh you are right, there was change with that :>